internal/service/alarms/internal/dictionary/dictionary.go

```diff
@@ -189,5 +205,9 @@ func getResourceTypes() []resourceType {
 			model:   resourceTypeCluster,
 			version: "4.16",
 		},
+		{
+			id:    4,
+			model: resourceTypeHub,
+		},
```

Comment:
I think is good but will not useful for now? To start with we are only just looking at managed clusters/SNO right...i.e specifically ignoring hub alerts? (we have mentioned this in design docs/epic i think)

Reply:
Sure

Reply:
done